### PR TITLE
[signing] Publish signing ceremony 2024-10-23

### PR DESCRIPTION
**Pull request by cfrantz**
Note: the release was aborted because a defect was discovered: The
`protect_when_primary` feature does not work as intended.  The defect
will be corrected by https://github.com/lowRISC/opentitan/pull/24879.

### CODE REVIEW

**Review by timothytrippel — APPROVED** [review]

**Review by moidx — APPROVED** [review]

**Comment by cfrantz** [comment]
@jettr found an error in ROM_EXT 0.5 that will be fixed by #24879.  Since the SiVAL and ProdA configurations do not currently define a flash configuration, they are not affected by this bug (but would be if they installed an ownership config that does define a flash config).

File: signing/logs/2024-10-23.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+# Signing Ceremony 2024-10-23
+
+- Purpose:
+    - Create new a ROM\_EXT release with ownership transfer and a validatable DICE chain.
+- Participants: cfrantz (leader), ttrippel (witness).
+
+## Ceremony Prolog
```
Comment (pqcfox) [inline signing/logs/2024-10-23.md:7]: [nit] `Prolog` -> `Prologue`

File: signing/logs/2024-10-23.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+  {
+    "command": "rsa-sign",
+    "result": {
+      "success": true
+    }
+  }
+]
+```
+
+## Ceremony Epilog
```
Comment (pqcfox) [inline signing/logs/2024-10-23.md:255]: [nit] `Epilog` -> `Epilogue`

**Review by pqcfox — APPROVED** [review]

**Review by sydgibs — APPROVED** [review]

**Review by pqcfox — APPROVED** [review]

**Comment by cfrantz** [comment]
I'm going to abort the release of ROM_EXT 0.5 because of the `protect_when_primary` (active) issue.  We'll sign a ROM_EXT 0.6 shortly that has the issue corrected.

I'm going to commit this PR, but only with the signing ceremony documentation; no binaries.

**Pull request merged**